build_test.sh

```diff
@@ -39,4 +39,6 @@ cd ../../test
 ./test_all.sh
 cd ..
 
+echo "\nRemoving test generated ext/oj/oj.so"
+rm ./ext/oj/oj.so
```

Comment:
Removing after running the tests means you can not run the test twice. This does not work for me.